Sources/MapboxMaps/Annotations/Generated/PointAnnotationManager.swift

```diff
@@ -91,6 +105,77 @@ public class PointAnnotationManager: AnnotationManagerInternal {
         displayLinkCoordinator.add(displayLinkParticipant)
     }
 
+    internal func createClustersLayers(clusterOptions: ClusterOptions) {
+        for index in 0..<clusterOptions.colorLevels.count {
+            let clusterLevelLayer = createClusterLevelLayer(level: index, clusterOptions: clusterOptions)
+            addClusterLayer(clusterLayer: clusterLevelLayer)
+        }
+        let clusterTextLayer = createClusterTextLayer(clusterOptions: clusterOptions)
+        addClusterLayer(clusterLayer: clusterTextLayer)
+    }
+
+    internal func createClusterLevelLayer(level: Int, clusterOptions: ClusterOptions) -> CircleLayer {
+        let layedID = "mapbox-iOS-cluster-circle-layer-" + String(level)
```

Comment:
I picked this to match the Android implementation, but happy to adjust: 
https://github.com/mapbox/mapbox-maps-android/blob/main/plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt#L267

Reply:
Either goes for me, but I'd suggest throwing the manager id here into the mix - as there might be layer id collisions in case of multiple annotation managers.